tests/cover/test_cover.py

```diff
@@ -0,0 +1,22 @@
+from src.cover.cover import Line, Cover
```

Comment:
```suggestion
from src.cover.cover import Cover
```
`Line` is not used I guess...